src/components/Message/MessageOptions.tsx

```diff
@@ -95,13 +95,14 @@ const UnMemoizedMessageOptions = <
           <ThreadIcon className='str-chat__message-action-icon' />
         </button>
       )}
-      {shouldShowReactions && (
-        <ReactionSelectorWithButton ReactionIcon={ReactionIcon} theme={theme} />
-      )}
+      {shouldShowReactions && <ReactionSelectorWithButton ReactionIcon={ReactionIcon} />}
     </div>
   );
 };
 
+/**
+ * @deprecated Consider moving to `MessageActions` from `stream-chat-react/unstable` instead.
+ */
```

Comment:
I'm not sure whether we should deprecate these components right now. Perhaps we do this after we gather the initial feedback?